### ISSUE TITLE
Shows zero notifications

### ISSUE THREAD

**Issue by zbs** [fabiospampinato/vscode-github-notifications-bell#23]
The bell indicates there are 0 notifications, but when I open my notifications page, there are several. I've provided my OAuth token according to the instructions, but GitHub tells me that the token is never used.

**Comment by fabiospampinato** [comment]
Yeah sorry I named the token wrong, it's a PAT token not an OAuth token (https://github.com/fabiospampinato/vscode-github-notifications-bell/issues/20), I haven't updated the readme yet 🤦‍♂️

Does that address the issue?

**Comment by zbs** [comment]
Does not seem to work with PAT either. To be clear, I'm using
```
githubNotificationsBell.oauthToken
```
I'm also using an enterprise GitHub domain, which I've reflected in the `githubNotificationsBell.domai` url

**Comment by fabiospampinato** [comment]
🤔 It's probably some bug related to the custom domain then I guess, PR welcome if you can find what the issue is.

**Comment by wdanilo** [comment]
It doesn't work for me either. I have public repositories and it shows 0 notifications.

**Comment by fabiospampinato** [comment]
Can you guys check if v3 fixed the issue?

**Comment by o-hushcha** [comment]
Works for me on v3!
